### PR TITLE
Build consistency fixes

### PR DESCRIPTION
**Pull request by bghgary**

### CODE REVIEW

**Comment by CedricGuillemet** [comment]
Fixed bgfx/bx flag with https://github.com/BabylonJS/bgfx.cmake/pull/68

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,75 +1,83 @@
...
+FetchContent_MakeAvailable(CMakeExtensions)
 
 if(IOS)
-    FetchContent_MakeAvailable(ios-cmake)
+    FetchContent_MakeAvailable_With_Message(ios-cmake)
     set(CMAKE_TOOLCHAIN_FILE "${ios-cmake_SOURCE_DIR}/ios.toolchain.cmake" CACHE PATH "")
     set(PLATFORM "OS64COMBINED" CACHE STRING "")
-    set(DEPLOYMENT_TARGET "12" CACHE STRING "")
+    set(DEPLOYMENT_TARGET "13" CACHE STRING "")
     set(ENABLE_ARC OFF CACHE STRING "Enables or disables ARC support.")
```
Comment (CedricGuillemet) [inline CMakeLists.txt:59]: When this line is removed, there is this build error :
```
/Users/runner/work/1/s/buildiOS/_deps/bgfx.cmake-src/bgfx/src/renderer_mtl.h:983:5: error: ARC forbids explicit message send of 'release'
                                MTL_RELEASE(m_ptrMips[ii]);
                                ^           ~~~~~~~~~~~~~
/Users/runner/work/1/s/buildiOS/_deps/bgfx.cmake-src/bgfx/src/renderer_mtl.h:734:9: note: expanded from macro 'MTL_RELEASE'
```

Comment (ryantrem) [reply CMakeLists.txt:59]: This is saying ARC should be off by default, and sub projects have to explicitly enable it if they want it, is that right?

Comment (CedricGuillemet) [reply CMakeLists.txt:59]: yes, per target, specify arc use with compilation flag `-fobjc-arc`

**Comment by bghgary** [comment]
@CedricGuillemet I can't approve my own PR, but I reviewed the changes you made and it looks good to me.

File: Install/Install.cmake [hunk 2]
```diff
@@ -159,5 +167,5 @@ endif()
 
 if(TARGET XMLHttpRequest)
     install_targets(XMLHttpRequest)
-    install_include(${jsruntimehost_SOURCE_DIR}/Polyfills/XMLHttpRequest/Include/Babylon)
+    install_include_for_target(XMLHttpRequest "Babylon")
 endif()
```
Comment (PolygonalSun) [inline Install/Install.cmake:171]: Nit: newline

**Pull request merged**